packages/rest-hooks/src/resource/SimpleRecord.ts

```diff
@@ -1,15 +1,15 @@
-import { AbstractInstanceType } from 'rest-hooks/types';
+import { EntityInstance } from 'rest-hooks/types';
```

Comment:
this is the same thing, but wrapped in Readonly<>. Arguments are never modified in these methods, and this allows us to not pay attention to private members.